core/taskengine/vm_runner_contract_write.go

```diff
@@ -123,31 +123,33 @@ func (r *ContractWriteProcessor) Execute(stepID string, node *avsproto.ContractW
 				PaymasterAndData:     common.Bytes2Hex(userOp.PaymasterAndData),
 				Signature:            common.Bytes2Hex(userOp.Signature),
 			},
-
-			TxReceipt: &avsproto.Evm_TransactionReceipt{
-				Hash:        txReceipt.TxHash.Hex(),
-				BlockHash:   txReceipt.BlockHash.Hex(),
-				BlockNumber: uint64(txReceipt.BlockNumber.Int64()),
-				// TODO: Need to fetch this, it isn't available
-				//From:            txReceipt.From.Hex(),
-				//To:              txReceipt.To.Hex(),
-				GasUsed:           txReceipt.GasUsed,
-				GasPrice:          uint64(txReceipt.EffectiveGasPrice.Int64()),
-				CumulativeGasUsed: txReceipt.CumulativeGasUsed,
-				// Fee:                 txReceipt.Fee,
-				ContractAddress: txReceipt.ContractAddress.Hex(),
-				Index:           uint64(txReceipt.TransactionIndex),
-				// TODO: convert raw log
-				//Logs:                txReceipt.Logs,
-				LogsBloom:    common.Bytes2Hex(bloom),
-				Root:         common.Bytes2Hex(txReceipt.PostState),
-				Status:       uint32(txReceipt.Status),
-				Type:         uint32(txReceipt.Type),
-				BlobGasPrice: blobGasPrice,
-				BlobGasUsed:  uint64(txReceipt.BlobGasUsed),
-			},
 		},
 	}
+	
+	if txReceipt != nil {
+		outputData.ContractWrite.TxReceipt = &avsproto.Evm_TransactionReceipt{
+			Hash:        txReceipt.TxHash.Hex(),
+			BlockHash:   txReceipt.BlockHash.Hex(),
+			BlockNumber: uint64(txReceipt.BlockNumber.Int64()),
+			// TODO: Need to fetch this, it isn't available
+			//From:            txReceipt.From.Hex(),
+			//To:              txReceipt.To.Hex(),
+			GasUsed:           txReceipt.GasUsed,
+			GasPrice:          uint64(txReceipt.EffectiveGasPrice.Int64()),
+			CumulativeGasUsed: txReceipt.CumulativeGasUsed,
+			// Fee:                 txReceipt.Fee,
+			ContractAddress: txReceipt.ContractAddress.Hex(),
+			Index:           uint64(txReceipt.TransactionIndex),
+			// TODO: convert raw log
+			//Logs:                txReceipt.Logs,
+			LogsBloom:    common.Bytes2Hex(bloom),
+			Root:         common.Bytes2Hex(txReceipt.PostState),
+			Status:       uint32(txReceipt.Status),
+			Type:         uint32(txReceipt.Type),
+			BlobGasPrice: blobGasPrice,
+			BlobGasUsed:  uint64(txReceipt.BlobGasUsed),
+		}
+	}
```

Comment:
This file should wait for the change in `https://github.com/AvaProtocol/EigenLayer-AVS/pull/199`